### PR TITLE
Sets up Travis CI.

### PR DESCRIPTION
**Pull request by damonkohler**

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -45,33 +45,17 @@ Getting Started without ROS
 
 On Ubuntu 14.04 (Trusty):
```
Comment (wohe) [inline docs/source/index.rst]: Should we add "and 16.04 (Xenial)? We have tested that it works before.

File: scripts/install_debs.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o verbose
+
+# Install the required libraries that are available as debs.
+apt-get update
```
Comment (wohe) [inline scripts/install_debs.sh]: sudo apt-get update?

**Comment by wohe** [comment]
LGTM

File: docs/source/index.rst (hunk 1 above)
Comment (damonkohler) [inline docs/source/index.rst]: I think we should wait to do so until we're testing Xenial in CI.

**Pull request merged**